fink_filters/filter_anomaly_notification/filter_utils.py

```diff
@@ -99,3 +99,40 @@ def msg_handler_tg(tg_data, channel_id, med):
                 timeout=8
             )
         time.sleep(3)
+
+
+def get_OID(ra, dec):
+    '''
+    Notes
+    ----------
+    The function determines the nearest ZTF DR OID by the given ra and dec.
+
+    Parameters
+    ----------
+    dec: float
+        Declination of candidate; J2000 [deg]
+    ra: float
+        Right Ascension of candidate; J2000 [deg]
+
+    Returns
+    -------
+        out: str
+            ZTF DR OID
+          or
+            None, if nothing was found
+    '''
+    url = "https://api.telegram.org/bot"
+    url += os.environ['ANOMALY_TG_TOKEN']
+    method = url + "/sendMessage"
```

Comment:
You should put the declaration of `url` and `method` in the `if` statement below (because they are undefined in the continuous integration environment).

Reply:
Understood, thank you, fixed it

Reply:
thanks -- this way the test can pass :-)